includes/ajax.php

```diff
@@ -476,10 +478,16 @@ function tsml_ajax_import() {
 			}
 			
 			//add custom meeting fields if available
-			foreach (array('types', 'group', 'data_source', 'venmo', 'conference_url', 'conference_phone') as $key) {
+			foreach (array('types', 'data_source', 'conference_url', 'conference_phone') as $key) {
 				if (!empty($meeting[$key])) add_post_meta($meeting_id, $key, $meeting[$key]);
 			}
 
+			// Add Group Id if applicable
+			if (!empty($meeting['group'])) {
+				$group_id = $groups[$meeting['group']];
+				if (!empty(group_id)) add_post_meta($meeting_id, 'group_id', $group_id);
```

Comment:
I'm very puzzled as to how this was missing earlier. Not having this basically means that if you imported from CSV that importing of any group information never worked? I thought it was just two fields that we were focusing on.

Reply:
I didn't go back and look prior to our last release, but we moved a bunch of fields into an array and looped through it. I thought maybe we introduced that but then?